README.md

```diff
@@ -56,14 +56,25 @@ Each user is allowed to have different local specifications: We don't put any re
 
 ##### What do I put in `user-config.yaml`?
 
-There's no "default" for `user-config.yaml` because it depends on system specifications and user preferences. Two things we do recommend keeping in `user-config.yaml` are the name of your Stata executable—which we'll prompt you to enter if necessary—and the location of a [SCons cache directory](http://scons.org/doc/2.0.1/HTML/scons-user/c4213.html)—if you have one. A Mac example where Example_User is running a factory-fresh StataMP and has local access to a directory named cache/template on Dropbox would be 
+There's no "default" for `user-config.yaml` because it depends on system specifications and user preferences. Three things we do recommend keeping in `user-config.yaml` are the name of your Stata executable, the location of a [SCons cache directory](http://scons.org/doc/2.0.1/HTML/scons-user/c4213.html), and the location of a release directory. These fields don't have to be specified if you're not using them, and we'll prompt you for their values at runtime if you've forgotten to specify them and they're necessary. A Mac example where Example_User is running a factory-fresh StataMP and has local access to a directories named cache/template and release on Dropbox would be 
 
 ```YAML
-stata_executable: statamp
+stata_executable: StataMP
```

Comment:
I would switch this back to lowercase. By default the Stata builder uses `statamp`. Bash is case-insensitive but no need to add confusion.